package.json

```diff
@@ -22,6 +22,8 @@
   },
   "homepage": "https://github.com/dgraph-io/typhoon-ui#readme",
   "dependencies": {
+    "@emotion/react": "^11.4.0",
+    "framer-motion": "^4.0.0",
```

Comment:
I think this shouldn't be here, since it's already there under `peerDependencies`.